apps/sla/tests/test_framework.py

```diff
@@ -552,3 +553,259 @@ def test_exclusion(self, component, excluded):
+                    },
+                    {},
+                    6,
+                ),
+                (
+                    {},
+                    {
+                        "impact": Impact.LOW,
+                    },
+                    6,
+                ),
+            ],
+        )
+        def test_latest_policy(
+            self, embargoed, is_community, flaw_kargs, affect_kargs, expected_slo
+        ):
+            slo_file = """
+---
+name: Major Incident
+description: SLO policy applied to approved Major Incident
+conditions:
+  affect:
+    - is not community
+  flaw:
+    - major incident state is major incident approved
+    - is not embargoed
+slo:
+  duration: 1
+  start:
+    latest:
+      flaw:
+        - major incident start date
+        - unembargo date
+      tracker:
+        - created date
+  type: business days
+
+---
+name: Exploit (KEV)
+description: SLO policy applied to approved Exploit (KEV)
+conditions:
+  affect:
+    - is not community
+  flaw:
+    - major incident state is exploits kev approved
+    - is not embargoed
+slo:
+  duration: 2
+  start:
+    latest:
+      flaw:
+        - major incident start date
+        - unembargo date
+      tracker:
+        - created date
+  type: business days
+
+---
+name: Critical
+description: SLO policy applied to critical impact
+conditions:
+  affect:
+    - aggregated impact is critical
+    - is not community
+  flaw:
+    - is not embargoed
+slo:
+  duration: 3
+  start:
+    latest:
+      flaw:
+        - unembargo date
+      tracker:
+        - created date
+  type: business days
+
+---
+name: Important
+description: SLO policy applied to important impact
+conditions:
+  affect:
+    - aggregated impact is important
+    - is not community
+  flaw:
+    - is not embargoed
+slo:
+  duration: 4
+  start:
+    latest:
+      flaw:
+        - unembargo date
+      tracker:
+        - created date
+  # when the deadline falls on Friday or weekend
+  # it is skipped and set to the following Monday
+  type: no week ending calendar days
+
+---
+name: Moderate
+description: SLO policy applied to moderate impact
+conditions:
+  affect:
+    - aggregated impact is moderate
+    - is not community
+  flaw:
+    - is not embargoed
+slo:
+  duration: 5
+  start:
+    latest:
+      flaw:
+        - unembargo date
+      tracker:
+        - created date
+  type: calendar days
+
+---
+name: Low
+description: SLO policy applied to low impact
+conditions:
+  affect:
+    - aggregated impact is low
+    - is not community
+  flaw:
+    - is not embargoed
+slo:
+  duration: 6
+  start:
+    latest:
+      flaw:
+        - unembargo date
+      tracker:
+        - created date
+  type: calendar days
+"""
+            if "major_incident_state" not in flaw_kargs:
+                MI = {"major_incident_state": Flaw.FlawMajorIncident.NOVALUE}
+            else:
+                MI = {}
+            flaw = FlawFactory(embargoed=embargoed, **flaw_kargs, **MI)
```

Comment:
```suggestion
            if "major_incident_state" not in flaw_kargs:
                flaw_kargs["major_incident_state"] =  Flaw.FlawMajorIncident.NOVALUE
            flaw = FlawFactory(embargoed=embargoed, **flaw_kargs)
```

Reply:
although I'd just prepopulate the test fixtures above (`flaw_kargs`) with `"major_incident_state": NOVALUE`